scripts/fmt.sh

```
@@ -18,6 +30,11 @@ fmt() {
     popd > /dev/null
 }
 
+# Check the versions of Noir and BB
+check_versions
+
+cd $SCRIPT_DIR/..
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)